ex2_hal/adcs/equipment_handler/inc/adcs_handler.h

```diff
@@ -481,6 +481,14 @@ typedef struct {
     uint16_t crc16_checksum;
 } adcs_file_info;
 
+typedef struct __attribute__((packed)){
+    uint8_t type;
+    uint8_t counter;
+    uint32_t size;
+    char file_name[20];
```

Comment:
is the file name guaranteed to never be greater than 20? There is a REDCONF_NAME_MAX you can use